### ISSUE TITLE
Ad-hoc jobs missing events/stdout

### ISSUE THREAD

**Issue by senorsmile** [ansible/awx#146]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
stdout capture is missing on nearly all jobs

##### ENVIRONMENT
* AWX version: awx_task:1.0.0.339
* Ansible version:  2.3.2
* Operating System: ubuntu 16.04
* Web Browser: chrome for mac

##### STEPS TO REPRODUCE

Using geerlingguy's role to deploy to a remote node: 
https://github.com/geerlingguy/ansible-role-awx

But with one patch (that shouldn't affect this):
https://github.com/geerlingguy/ansible-role-awx/pull/3

Running a "Command" against localhost (as well as several other hosts I have added to inventory) always shows the stdout capture is missing error.  

##### EXPECTED RESULTS
I should be seeing the actual standard out.  I can see it if I enter the container and: 

```
[root@awx job_status]# cat 16-72032534-980f-11e7-83d4-0242ac110006.out
Using /etc/ansible/ansible.cfg as config file
META: ran handlers
localhost | SUCCESS | rc=0 >>
total 8.0K
drwx------  2 root root 4.0K Sep 12 23:09 .
drwxrwxrwt 11 root root 4.0K Sep 12 23:09 ..

META: ran handlers
META: ran handlers
```
But the awx UI doesn't pick it up for any jobs (except for the built in hello world job!!!!)

##### ACTUAL RESULTS

<img width="648" alt="screen shot 2017-09-12 at 5 08 36 pm" src="https://user-images.githubusercontent.com/2666849/30353849-90035f2c-97dd-11e7-8485-60b50d89ba69.png">

##### ADDITIONAL INFORMATION

<img width="619" alt="screen shot 2017-09-12 at 5 08 27 pm" src="https://user-images.githubusercontent.com/2666849/30353862-9a5a758c-97dd-11e7-8eac-a740047e798b.png">

**Comment by matburt** [comment]
Our job detail UI is driven by callback event data (even though it looks like stdout). The stdout .out files are there mainly for when you click the download button.

It seems like the callback receiver may be having issues in your awx_task container... would it be possible to grab some logs from the container so we could see what's going on?

**Comment by senorsmile** [comment]
I unfortunately trashed the original VM.  I rebuilt it using the exact same playbook and tested that the problem is in fact still happening.  I have a feeling this was not happening on a previous build.  

I have downgraded from 
```1.0.0-339-gd9e50ab6d```
to
```1.0.0-331-g3118092e5 ```
to test further.  

I'll grab some logs from within the container once this build is done and db migrations have completed.

**Comment by senorsmile** [comment]
same with v331.  

also tried deploying from geerlingguy's master and got the exact same thing. 


The only interesting logs I found were supervisord.log in the container and the docker logs itself.  Anything else I should be looking for? 

```
[root@awx awx]# cat supervisord.log
2017-09-14 19:02:33,694 CRIT Supervisor running as root (no user in config file)
2017-09-14 19:02:33,694 WARN For [program:awx-celeryd-beat], redirect_stderr=true but stderr_logfile has also been set to a filename, the filename has been ignored
2017-09-14 19:02:33,706 INFO RPC interface 'supervisor' initialized
2017-09-14 19:02:33,706 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2017-09-14 19:02:33,707 INFO supervisord started with pid 90
2017-09-14 19:02:34,710 INFO spawned: 'celery' with pid 93
2017-09-14 19:02:34,712 INFO spawned: 'channels-worker' with pid 94
2017-09-14 19:02:34,716 INFO spawned: 'callback-receiver' with pid 95
2017-09-14 19:02:34,722 INFO spawned: 'awx-celeryd-beat' with pid 96
2017-09-14 19:02:35,729 INFO success: celery entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-09-14 19:02:35,729 INFO success: channels-worker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-09-14 19:02:35,729 INFO success: callback-receiver entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-09-14 19:02:35,730 INFO success: awx-celeryd-beat entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
```


```
Using /etc/ansible/ansible.cfg as config file
127.0.0.1 | SUCCESS => {
    "changed": true, 
    "db": "awx"
}
Operations to perform:
  Synchronize unmigrated apps: solo, api, staticfiles, messages, channels, django_extensions, ui, rest_framework, polymorphic
  Apply all migrations: sso, taggit, sessions, djcelery, sites, kombu_transport_django, social_auth, contenttypes, auth, conf, main
Synchronizing apps without migrations:
  Creating tables...
    Running deferred SQL...
  Installing custom SQL...
Running migrations:
  Rendering model states... DONE
  Applying contenttypes.0001_initial... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0001_initial... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying taggit.0001_initial... OK
  Applying taggit.0002_auto_20150616_2121... OK
  Applying main.0001_initial... OK
  Applying main.0002_squashed_v300_release... OK
  Applying main.0003_squashed_v300_v303_updates... OK
  Applying main.0004_squashed_v310_release... OK
  Applying conf.0001_initial... OK
  Applying conf.0002_v310_copy_tower_settings... OK
  Applying conf.0003_v310_JSONField_changes... OK
  Applying conf.0004_v320_reencrypt... OK
  Applying djcelery.0001_initial... OK
  Applying kombu_transport_django.0001_initial... OK
  Applying main.0005_squashed_v310_v313_updates... OK
  Applying main.0006_v320_release... OK
  Applying main.0007_v320_data_migrations... OK
  Applying main.0008_v320_drop_v1_credential_fields... OK
  Applying sessions.0001_initial... OK
  Applying sites.0001_initial... OK
  Applying social_auth.0001_initial... OK
  Applying social_auth.0002_add_related_name... OK
  Applying social_auth.0003_alter_email_max_length... OK
  Applying social_auth.0004_auto_20160423_0400... OK
  Applying social_auth.0005_auto_20160727_2333... OK
  Applying sso.0001_initial... OK
  Applying sso.0002_expand_provider_options... OK
>>> <User: admin>
>>> Default organization added.
Demo Credential, Inventory, and Job Template added.
Successfully registered instance awx
(changed: True)
Creating instance group tower
Added instance awx to tower
(changed: True)
2017-09-14 19:02:33,694 CRIT Supervisor running as root (no user in config file)
2017-09-14 19:02:33,694 WARN For [program:awx-celeryd-beat], redirect_stderr=true but stderr_logfile has also been set to a filename, the filename has been ignored
2017-09-14 19:02:33,706 INFO RPC interface 'supervisor' initialized
2017-09-14 19:02:33,706 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2017-09-14 19:02:33,707 INFO supervisord started with pid 90
2017-09-14 19:02:34,710 INFO spawned: 'celery' with pid 93
2017-09-14 19:02:34,712 INFO spawned: 'channels-worker' with pid 94
2017-09-14 19:02:34,716 INFO spawned: 'callback-receiver' with pid 95
2017-09-14 19:02:34,722 INFO spawned: 'awx-celeryd-beat' with pid 96
2017-09-14 19:02:35,729 INFO success: celery entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-09-14 19:02:35,729 INFO success: channels-worker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-09-14 19:02:35,729 INFO success: callback-receiver entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-09-14 19:02:35,730 INFO success: awx-celeryd-beat entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-09-14 19:02:46,657 WARNING  py.warnings /var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/management/base.py:260: RemovedInDjango19Warning: "requires_model_validation" is deprecated in favor of "requires_system_checks".
  RemovedInDjango19Warning)

[2017-09-14 19:02:46,695: DEBUG/MainProcess] Setting default socket timeout to 30
[2017-09-14 19:02:46,695: INFO/MainProcess] beat: Starting...
[2017-09-14 19:02:46,742: DEBUG/MainProcess] Current schedule:
<Entry: purge_stdout_files awx.main.tasks.purge_old_stdout_files() <freq: 7.00 days>
<Entry: cluster_heartbeat awx.main.tasks.cluster_node_heartbeat() <freq: 1.00 minute>
<Entry: authtoken_cleanup awx.main.tasks.cleanup_authtokens() <freq: 30.00 days>
<Entry: task_manager awx.main.scheduler.tasks.run_task_manager() <freq: 20.00 seconds>
<Entry: tower_scheduler awx.main.tasks.awx_periodic_scheduler() <freq: 30.00 seconds>
<Entry: celery.backend_cleanup celery.backend_cleanup() <crontab: 0 4 * * * (m/h/d/dM/MY)>
<Entry: admin_checks awx.main.tasks.run_administrative_checks() <freq: 30.00 days>
[2017-09-14 19:02:46,743: DEBUG/MainProcess] beat: Ticking with max interval->1.00 minute
[2017-09-14 19:02:46,761: DEBUG/MainProcess] Start from server, version: 0.9, properties: {u'information': u'Licensed under the MPL.  See http://www.rabbitmq.com/', u'product': u'RabbitMQ', u'copyright': u'Copyright (C) 2007-2017 Pivotal Software, Inc.', u'capabilities': {u'exchange_exchange_bindings': True, u'connection.blocked': True, u'authentication_failure_close': True, u'direct_reply_to': True, u'basic.nack': True, u'per_consumer_qos': True, u'consumer_priorities': True, u'consumer_cancel_notify': True, u'publisher_confirms': True}, u'cluster_name': u'rabbit@f4df6155a760', u'platform': u'Erlang/OTP 19.2.1', u'version': u'3.6.12'}, mechanisms: [u'PLAIN', u'AMQPLAIN'], locales: [u'en_US']
[2017-09-14 19:02:46,763: DEBUG/MainProcess] Open OK!
[2017-09-14 19:02:46,767: DEBUG/MainProcess] beat: Waking up in 19.95 seconds.
[2017-09-14 19:03:06,739: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:03:06,745: DEBUG/MainProcess] using channel_id: 1
[2017-09-14 19:03:06,746: DEBUG/MainProcess] Channel open
[2017-09-14 19:03:06,759: DEBUG/MainProcess] beat: Synchronizing schedule...
[2017-09-14 19:03:06,763: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->e65c5303-b860-4230-872d-22a6ed51379f
[2017-09-14 19:03:06,764: DEBUG/MainProcess] beat: Waking up in 9.98 seconds.
[2017-09-14 19:03:16,756: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:03:16,770: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->f8927957-bda1-4d55-b219-f167fb62e060
[2017-09-14 19:03:16,771: DEBUG/MainProcess] beat: Waking up in 9.97 seconds.
[2017-09-14 19:03:26,756: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:03:26,785: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->0b2bb5ea-e69f-47d2-8a8f-2d9dd789776e
[2017-09-14 19:03:26,788: DEBUG/MainProcess] beat: Waking up in 19.96 seconds.
[2017-09-14 19:03:46,767: INFO/MainProcess] Scheduler: Sending due task cluster_heartbeat (awx.main.tasks.cluster_node_heartbeat)
[2017-09-14 19:03:46,789: DEBUG/MainProcess] awx.main.tasks.cluster_node_heartbeat sent. id->516f812f-0c15-4966-bd13-99bb24a25ae3
[2017-09-14 19:03:46,793: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:03:46,853: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->beb8c89d-18bb-49ab-89dc-df8ccc42b4e3
[2017-09-14 19:03:46,853: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:03:46,876: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->7ceb67af-397e-4c8d-89dc-0f024ceabee6
[2017-09-14 19:03:46,878: DEBUG/MainProcess] beat: Waking up in 19.99 seconds.
[2017-09-14 19:04:06,893: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:04:06,940: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->79cdc81a-6b47-4a96-bc50-ec7de6f38927
[2017-09-14 19:04:06,943: DEBUG/MainProcess] beat: Waking up in 9.91 seconds.
[2017-09-14 19:04:16,861: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:04:16,864: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->6d664056-6154-42b7-be7a-bfa9cc03635b
[2017-09-14 19:04:16,868: DEBUG/MainProcess] beat: Waking up in 10.02 seconds.
[2017-09-14 19:04:26,911: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:04:26,913: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->944d751e-32f9-42fb-a06d-604d3a75c2eb
[2017-09-14 19:04:26,916: DEBUG/MainProcess] beat: Waking up in 19.85 seconds.
[2017-09-14 19:04:46,769: INFO/MainProcess] Scheduler: Sending due task cluster_heartbeat (awx.main.tasks.cluster_node_heartbeat)
[2017-09-14 19:04:46,775: DEBUG/MainProcess] awx.main.tasks.cluster_node_heartbeat sent. id->211891a1-ba93-4659-b0c3-7b9e1fb2ed14
[2017-09-14 19:04:46,785: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:04:46,863: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:04:46,876: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->64b4c00f-b0db-4cd2-b65f-224197381d0a
[2017-09-14 19:04:46,877: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:04:46,918: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:04:46,923: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->c10808ea-4500-402f-8b4d-aa4ba0875745
[2017-09-14 19:04:46,924: DEBUG/MainProcess] beat: Waking up in 19.99 seconds.
[2017-09-14 19:05:06,946: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:05:06,948: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->fc46a12a-2b1a-4dae-98e2-52e1d8b460ac
[2017-09-14 19:05:06,951: DEBUG/MainProcess] beat: Waking up in 9.90 seconds.
[2017-09-14 19:05:16,866: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:05:16,868: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->b0af73e6-20e1-4074-8091-f5fd54a35aa7
[2017-09-14 19:05:16,870: DEBUG/MainProcess] beat: Waking up in 10.07 seconds.
[2017-09-14 19:05:26,953: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:05:26,956: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->d3723149-2b01-4097-add2-c8f1936af207
[2017-09-14 19:05:26,966: DEBUG/MainProcess] beat: Waking up in 19.81 seconds.
[2017-09-14 19:05:46,795: INFO/MainProcess] Scheduler: Sending due task cluster_heartbeat (awx.main.tasks.cluster_node_heartbeat)
[2017-09-14 19:05:46,797: DEBUG/MainProcess] awx.main.tasks.cluster_node_heartbeat sent. id->eb541aeb-167e-476b-9f4f-fead41acc2e6
[2017-09-14 19:05:46,798: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:05:46,868: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:05:46,881: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->a40aa029-5f67-4f46-b2ed-e92ae8d64dc6
[2017-09-14 19:05:46,882: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:05:46,960: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:05:46,965: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->93e79e06-2488-416e-901d-b6cffb3a75de
[2017-09-14 19:05:46,966: DEBUG/MainProcess] beat: Waking up in 19.99 seconds.
[2017-09-14 19:06:06,976: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:06:06,983: DEBUG/MainProcess] beat: Synchronizing schedule...
[2017-09-14 19:06:06,996: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->10a6f0fc-cdc1-4478-bd9a-c02309a858e6
[2017-09-14 19:06:06,997: DEBUG/MainProcess] beat: Waking up in 9.86 seconds.
[2017-09-14 19:06:16,870: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:06:16,883: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->00d1b67e-b3a1-424b-80bd-365f5e7aec66
[2017-09-14 19:06:16,886: DEBUG/MainProcess] beat: Waking up in 10.08 seconds.
[2017-09-14 19:06:26,977: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:06:26,990: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->e2ff950a-3aff-4376-8734-1762c1a33c75
[2017-09-14 19:06:26,992: DEBUG/MainProcess] beat: Waking up in 19.79 seconds.
[2017-09-14 19:06:46,800: INFO/MainProcess] Scheduler: Sending due task cluster_heartbeat (awx.main.tasks.cluster_node_heartbeat)
[2017-09-14 19:06:46,802: DEBUG/MainProcess] awx.main.tasks.cluster_node_heartbeat sent. id->02585bef-f4dd-49a0-80a1-f253a20eba85
[2017-09-14 19:06:46,804: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:06:46,874: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:06:46,883: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->a75b6000-1a23-4f34-a067-01e0ab15640c
[2017-09-14 19:06:46,886: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:06:46,979: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:06:46,991: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->b36f12a6-2501-4335-8309-cbe5a5726b63
[2017-09-14 19:06:46,995: DEBUG/MainProcess] beat: Waking up in 19.99 seconds.
[2017-09-14 19:07:07,008: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:07:07,010: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->134d2f2d-2f9a-496d-a823-b882d75cedf8
[2017-09-14 19:07:07,012: DEBUG/MainProcess] beat: Waking up in 9.86 seconds.
[2017-09-14 19:07:16,881: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:07:16,883: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->c04bff01-276b-41d4-b19e-632147ff6f47
[2017-09-14 19:07:16,887: DEBUG/MainProcess] beat: Waking up in 10.11 seconds.
[2017-09-14 19:07:27,014: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:07:27,016: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->07a292ef-601b-4c7c-a9cd-096109a65dad
[2017-09-14 19:07:27,019: DEBUG/MainProcess] beat: Waking up in 19.77 seconds.
[2017-09-14 19:07:46,813: INFO/MainProcess] Scheduler: Sending due task cluster_heartbeat (awx.main.tasks.cluster_node_heartbeat)
[2017-09-14 19:07:46,826: DEBUG/MainProcess] awx.main.tasks.cluster_node_heartbeat sent. id->d3024d5d-beeb-4f49-9820-a86916a49540
[2017-09-14 19:07:46,829: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:07:46,898: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:07:46,902: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->f5f13c39-e2d2-4a9b-a36c-c0ab674caf89
[2017-09-14 19:07:46,904: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:07:47,015: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:07:47,028: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->260d6fd7-e109-44bd-aace-fdb4fc12b416
[2017-09-14 19:07:47,031: DEBUG/MainProcess] beat: Waking up in 19.99 seconds.
[2017-09-14 19:08:07,045: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:08:07,058: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->7cc59d35-2832-478a-a6c5-79f86bfb654c
[2017-09-14 19:08:07,059: DEBUG/MainProcess] beat: Waking up in 9.84 seconds.
[2017-09-14 19:08:16,912: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:08:16,913: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->2baf3bc8-f263-48ea-97a0-b5b6ee054b91
[2017-09-14 19:08:16,914: DEBUG/MainProcess] beat: Waking up in 10.12 seconds.
[2017-09-14 19:08:27,051: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:08:27,053: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->9c27113a-b857-4d9e-a6e1-d6ac5800a783
[2017-09-14 19:08:27,054: DEBUG/MainProcess] beat: Waking up in 19.75 seconds.
[2017-09-14 19:08:46,817: INFO/MainProcess] Scheduler: Sending due task cluster_heartbeat (awx.main.tasks.cluster_node_heartbeat)
[2017-09-14 19:08:46,828: DEBUG/MainProcess] awx.main.tasks.cluster_node_heartbeat sent. id->e0fff4da-d39a-4920-965c-12e89aaa4e29
[2017-09-14 19:08:46,835: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:08:46,922: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:08:46,927: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->23040841-4560-4cca-acfc-60f0f167b98e
[2017-09-14 19:08:46,932: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:08:47,053: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:08:47,066: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->c852c092-a115-4aff-8ce1-81e16570efc4
[2017-09-14 19:08:47,068: DEBUG/MainProcess] beat: Waking up in 19.99 seconds.
[2017-09-14 19:09:07,082: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:09:07,085: DEBUG/MainProcess] beat: Synchronizing schedule...
[2017-09-14 19:09:07,090: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->96d1f648-3bf2-49e9-92c5-ac5d6bad88a9
[2017-09-14 19:09:07,091: DEBUG/MainProcess] beat: Waking up in 9.83 seconds.
[2017-09-14 19:09:16,934: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:09:16,936: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->b8a27e12-e2d0-4a42-8ada-b67f55d4a95c
[2017-09-14 19:09:16,939: DEBUG/MainProcess] beat: Waking up in 10.14 seconds.
[2017-09-14 19:09:27,084: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:09:27,086: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->398c64c9-49b8-43b1-8467-6fc8d2fb20f5
[2017-09-14 19:09:27,089: DEBUG/MainProcess] beat: Waking up in 19.72 seconds.
[2017-09-14 19:09:46,830: INFO/MainProcess] Scheduler: Sending due task cluster_heartbeat (awx.main.tasks.cluster_node_heartbeat)
[2017-09-14 19:09:46,833: DEBUG/MainProcess] awx.main.tasks.cluster_node_heartbeat sent. id->3aec4ce7-d9fe-47dd-ac4b-86441195e509
[2017-09-14 19:09:46,836: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:09:46,935: INFO/MainProcess] Scheduler: Sending due task tower_scheduler (awx.main.tasks.awx_periodic_scheduler)
[2017-09-14 19:09:46,947: DEBUG/MainProcess] awx.main.tasks.awx_periodic_scheduler sent. id->a3878ab0-1df8-4572-875b-5c3d730f7cd4
[2017-09-14 19:09:46,949: DEBUG/MainProcess] beat: Waking up now.
[2017-09-14 19:09:47,091: INFO/MainProcess] Scheduler: Sending due task task_manager (awx.main.scheduler.tasks.run_task_manager)
[2017-09-14 19:09:47,096: DEBUG/MainProcess] awx.main.scheduler.tasks.run_task_manager sent. id->fee6b827-51ec-4827-b6db-f8f814831d9f
[2017-09-14 19:09:47,097: DEBUG/MainProcess] beat: Waking up in 19.99 seconds.
```

**Comment by matburt** [comment]
It looks like the celery worker is not able to actually pick up jobs submitted to it. This seems like it's probably an issue with the linked rabbitmq container.

I don't see a broker connection stanza either... is your rabbitmq container running? are there any errors from it?

**Comment by senorsmile** [comment]
Rabbitmq is up.  

docker logs: 
https://pastebin.com/tB8BUqkY

In the previously pasted log, wouldn't this be the rabbitmq connection:

```
[2017-09-14 19:02:46,761: DEBUG/MainProcess] Start from server, version: 0.9, properties: {u'information': u'Licensed under the MPL.  See http://www.rabbitmq.com/', u'product': u'RabbitMQ', u'copyright': u'Copyright (C) 2007-2017 Pivotal Software, Inc.', u'capabilities': {u'exchange_exchange_bindings': True, u'connection.blocked': True, u'authentication_failure_close': True, u'direct_reply_to': True, u'basic.nack': True, u'per_consumer_qos': True, u'consumer_priorities': True, u'consumer_cancel_notify': True, u'publisher_confirms': True}, u'cluster_name': u'rabbit@f4df6155a760', u'platform': u'Erlang/OTP 19.2.1', u'version': u'3.6.12'}, mechanisms: [u'PLAIN', u'AMQPLAIN'], locales: [u'en_US']
```

I should repeat that there are in fact a couple of jobs that DO output stdout as expected: 
- the "Demo Project" SCM Update
- the "Demo Job Template" hello world playbook

However, all other jobs get the error, including running an ad_hoc command or ping against any nodes, including the default localhost inventory item.

**Comment by senorsmile** [comment]
I just tried a completely different installation method, using geerlingguy's prebuilt container images.  I used docker-compose and brought them up on a completely different machine. 

Direction here:
https://www.jeffgeerling.com/blog/2017/get-started-using-ansible-awx-open-source-tower-version-one-minute

I am getting the EXACT same problems.  All ad_hoc commands to localhost or any other hosts I add to inventory show  the 'stdout capture is missing'.

**Comment by senorsmile** [comment]
Correction... now all playbooks are displaying properly.     I cannot reproduce playbooks giving any errors.  It's only the ad hoc "Run Commands" jobs that display the error.

I had a colleague who is running awx on Fedora try, and ad hoc commands are doing the same thing for him.

**Comment by matburt** [comment]
I'm definitely having trouble reproducing this... are the awx containers having trouble getting to the rabbitmq container?

**Comment by senorsmile** [comment]
The containers are definitely able to contact each other.  

Here are the directions I sent to my colleague who reproduced the error: 

- go to inventories
- click Demo Inventory
- click Hosts button towards the top
- put a check mark to the node
- click run commands
- select command module
- set arguments to anything (e.g. ls -alh)
- click launch

**Comment by matburt** [comment]
Is it just ad-hoc commands? Normal jobs run okay?

**Comment by senorsmile** [comment]
In my original build it was both (although I now think I may have broken something else in the first build that I wiped). 

In my last 5 builds, with geerlingguy's prebuilt docker images normal jobs do run okay.  So it is just ad-hoc commands that seem to be consistently an issue.

**Comment by moralloki** [comment]
Just adding my data to this.  I see the same results; stdout capture is missing.  This is only happening for ad-hoc commands.

ENVIRONMENT
- AWX version: awx_task:1.0.0.282
- Ansible version: 2.3.1.0
- Operating System: Fedora 26
- Web Browser: chrome for fedora

**Comment by matburt** [comment]
Okay, so this is one of the architectural gaps between how we deploy AWX and Tower itself, there are some major changes coming to how stdout is displayed for ad-hoc jobs to make it event driven like normal jobs.

In the meantime, you can get the details of the ad-hoc command by looking at the API endpoint for the ad-hoc command: `/api/v2/ad_hoc_commands/7/events/`

This will show you the actual ansible event level data as well as stdout/stderr (as a property of the event that would have been responsible for emitting it.

**Comment by matburt** [comment]
I'm going to close this issue as we'll be covering it here:

https://github.com/ansible/awx/issues/200
